tests/test_data_store_merge_operations.py

```diff
@@ -215,7 +218,9 @@ def test_merge_platforms_with_different_sensor_names(self):
             assert len(states_before_merge) == 0
             assert len(contacts_before_merge) == 0
 
-            self.store.merge_platforms([platform_2.platform_id], platform.platform_id)
+            self.store.merge_platforms(
```

Comment:
Most of the other calls to merge_platforms seem to have been switched to merge_generic, apart from this one and a couple of others.